src/AdminPage/components/SuperUserForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { reduxForm, Field } from "redux-form";
+import axios from "axios";
+import UserList from "./UserList";
+import { bindActionCreators } from "redux";
+import "../css/navbar.css";
+import "../css/modal.css";
+import { fetchUserInfo, updateToSuperUser, removeSuperUserStatus} from "../../actions/index";
+
+
+class SuperUserForm extends Component {
+    constructor(props) {
+        super(props);
+        
+      }
+
+      
+      updateSuperUser = (event, id) => {
+        event.preventDefault();
+        if (this.props.isSuperUser === true){
+          this.props.removeSuperUserStatus(id);
+        } else {
+          this.props.updateToSuperUser(id);
+        }
+        setTimeout(function() {
```

Comment:
Is this because the status isn't updating? Forcing a reload is not the only way to update this component